examples/channel_inversion/model_config.py

```diff
@@ -41,12 +41,14 @@ def construct_solver(store_station_time_series=True, **model_options):
     options.simulation_export_time = t_export
     options.simulation_end_time = t_end
     options.horizontal_velocity_scale = u_mag
-    options.output_directory = 'outputs_forward'
+    options.output_directory = f'{pwd}/outputs_forward'
     options.fields_to_export = ['uv_2d', 'elev_2d']
     options.swe_timestepper_type = 'CrankNicolson'
     if not hasattr(options.swe_timestepper_options, 'use_automatic_timestep'):
         options.timestep = dt
     options.update(model_options)
+    if not options.no_exports:
+        File(f'{pwd}/manning.pvd').write(manning_2d)
```

Comment:
Oh, I think I forgot this in by mistake. Should be removed.

Reply:
Done